fastcrypto-derive/src/lib.rs

```diff
@@ -38,3 +38,40 @@ pub fn silent_debug(source: TokenStream) -> TokenStream {
     };
     gen.into()
 }
+
+/// Extend implementations of Add, Sub, Mul and Neg into implementations of Add, Sub, Neg, AddAssign,
+/// SubAssign and MulAssign for all combinations of borrowed and owner inputs.
```

Comment:
This will allow the macro to be used to implement combinations of borrowed and owned even when derive_more cannot be used.